test/common/protobuf/utility_test.cc

```diff
@@ -1571,6 +1571,21 @@ TEST_P(DeprecatedFieldsTest, DEPRECATED_FEATURE_TEST(IndividualFieldDeprecated))
   EXPECT_EQ(1, deprecated_feature_seen_since_process_start_.value());
 }
 
+TEST_P(DeprecatedFieldsTest, DEPRECATED_FEATURE_TEST(IndividualFieldDeprecatedEmitsCrash)) {
```

Comment:
Does this need to be a `DEPRECATED_FEATURE_TEST`? I think it would run time in the fatal-by-default build mode (since you are basically doing this at runtime).

Reply:
Removed.